objects/image.py

```diff
@@ -334,6 +334,7 @@ def matches_dashboard(self, dashboard_file):
 							["architectures"]
 			archs_from_dash.sort()
 			self.archs.sort()
+			print(self.archs == archs_from_dash)
```

Comment:
```suggestion
```
This looks like it was made for debugging purposes. If it is, can you remove it?